setup.pl

```diff
@@ -299,9 +299,14 @@
 		$crypt = $ENV{'crypt'};
 		}
 	else {
+        system("stty -echo");
```

Comment:
Can you fix the indents here?

Reply:
Indent fixed - https://github.com/webmin/webmin/commit/defa41f0b3098f742062b126bd1c7099a11f20af.